librz/util/print.c

```diff
@@ -1245,8 +1245,17 @@ RZ_API void rz_print_bytes(RzPrint *p, const ut8 *buf, int len, const char *fmt)
 	}
 }
 
-RZ_API void rz_print_raw(RzPrint *p, ut64 addr, const ut8 *buf, int len) {
-	p->write(buf, len);
+RZ_API void rz_print_raw(RzPrint *p, ut64 addr, const ut8 *buf, int len, bool stop_at_null) {
+	if (stop_at_null) {
+		for (int i = 0; i < len; i++) {
+			if (buf[i] == '\0') {
+				break;
+			}
+			p->write(&buf[i], 1);
+		}
```

Comment:
this is ok, but is much more faster to call `rz_str_nlen` to get the actual len and then call the write.